plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of Nomad server.
+      required: true
+      type: str
+    use_ssl:
+      description:
+        - Use TLS/SSL connection.
+      type: bool
+      default: false
+    timeout:
+      description:
+        - Timeout (in seconds) for the request to Nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - Enable TLS/SSL certificate validation.
+      type: bool
+      default: true
+    client_cert:
+      description:
+        - Path of certificate for TLS/SSL.
+      type: path
+    client_key:
+      description:
+        - Path of certificate's private key for TLS/SSL.
+      type: path
+    namespace:
+      description:
+        - Namespace for Nomad.
+      type: str
+    token:
+      description:
+        - ACL token for authentification.
+      type: str
+    name:
+      description:
+        - Name of job for delete,stop and start job without source.
+        - Name of job for delete, stop and start job without source.
+        - Either this or I(content) must be specified.
+      type: str
+    state:
+      description:
+        - Deploy or remove job.
+      choices: ["present", "absent"]
+      required: true
+      type: str
+    force_start:
+      description:
+        - Force job to started.
+      type: bool
+      default: false
+    content:
+      description:
+        - Content of Nomad job.
+        - Either this or I(name) must be specified.
+      type: str
+    content_format:
+      description:
+        - Type of content of Nomad job.
+      choices: ["hcl", "json"]
+      default: hcl
+      type: str
+'''
+
+EXAMPLES = '''
+- name: Create job
+  community.general.nomad_job:
+    host: localhost
+    state: present
+    content: "{{ lookup('ansible.builtin.file', 'job.hcl') }}"
+    timeout: 120
+- name: Stop job
+  community.general.nomad_job:
+    host: localhost
+    state: absent
+    name: api
+- name: Force job to start
+  community.general.nomad_job:
+    host: localhost
+    state: present
+    name: api
+    timeout: 120
+    force_start: true
+'''
+
+
+import os
+import json
+
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils._text import to_native
+
+import_nomad = None
+try:
+    import nomad
```

Comment:
```suggestion
    import nomad
```
if this is not embedded to standard python distribution, it must be put in `requirements:` doc section

Reply:
This has not yet been resolved.

Reply:
Sorry, it was. I confused tabs and looked into the code of the wrong PR... -.-